tests/tests_pytorch/loggers/test_logger.py

```diff
@@ -257,6 +257,18 @@ def test_dummylogger_noop_method_calls():
     logger.log_metrics("1", 2, three="three")
 
 
+def test_dummlogger_arbitrary_method_calls(caplog):
+    """Test that the DummyLogger can be called with non existing methods."""
+    logger = DummyLogger()
+    # Example method from WandbLogger
+    logger.log_text("1", columns=["1", "2", "3"], data=[[1, 2, 3]])
```

Comment:
Since getattr was implemented, you could now assert

```py
assert hasattr(logger, "cocofruit")
assert callable(logger.cocofruit)
```

Reply:
Yes, that looks more clean!